docs/reference/redirects.asciidoc

```diff
@@ -527,4 +527,48 @@ See <<cluster-shard-allocation-filtering>>.
 [role="exclude",id="misc-cluster"]
 === Miscellaneous cluster settings
 
-See <<misc-cluster-settings>>.
\ No newline at end of file
+See <<misc-cluster-settings>>.
+
+[role="exclude",id="modules"]
+=== Modules
+
+This page has been removed.
+
+For circuit breaker settings, see <<circuit-breaker>>.
```

Comment:
I'd probably pare this down to a simple list instead of repeating "For xyz settings" for each xref. Something like:

```
See <<settings, Configuring Elasticsearch>> for settings information:

* <<modules-cluster>>
* <<modules-dicsovery-settings>>
...
```

OTOH, this isn't a high-traffic page, so...leaving it as-is is fine, too.

Reply:
This is a good idea and much more readable IMO. Updated with 3068f21. Thanks for the suggestion!